### PR TITLE
Add PKO boilerplate for HypershiftLogForwarder CRD

### PR DESCRIPTION
**Pull request by hectorakemp**
Fixes https://issues.redhat.com/browse/OSD-18379

### CODE REVIEW

**Comment by hectorakemp** [comment]
Once we can get a staging MC up, then we can test this @bmeng @samanthajayasinghe. How to test: 
```
podman build -t quay.io/hkemp/hypershift-logging-operator-package-template:latest -f  ~/redhat/go/hypershift-logging-operator/hack/hypershift/package/hypershift-logging-operator.Containerfile . 
podman push quay.io/hkemp/hypershift-logging-operator-package-template:latest
ocm backplane login $mc_id
oc project $hcp_ns
oc --as backplane-cluster-admin apply -f - <<EOF
apiVersion: package-operator.run/v1alpha1
kind: Package
metadata:
  name: hypershift-logging-operator
spec:
  image: quay.io/hkemp/hypershift-logging-operator-package-template:latest
EOF
```

File: deploy/crds/logging.managed.openshift.io_hypershiftlogforwarders.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.10.0
+    # see deploy/package/manifest.yaml
+    package-operator.run/phase: hlf
```
Comment (bmeng) [inline deploy/crds/logging.managed.openshift.io_hypershiftlogforwarders.yaml:8]: should we use full name here or the short version?

**Pull request merged**